SocialCareCaseViewerApi/V1/Controllers/BaseController.cs

```diff
@@ -1,29 +0,0 @@
-using Microsoft.AspNetCore.Mvc;
-using Newtonsoft.Json;
-using Newtonsoft.Json.Serialization;
-
-namespace SocialCareCaseViewerApi.V1.Controllers
-{
-    public class BaseController : Controller
-    {
-        public BaseController()
-        {
-            ConfigureJsonSerializer();
-        }
-
-        public static void ConfigureJsonSerializer()
-        {
-            JsonConvert.DefaultSettings = () =>
-            {
-                var settings = new JsonSerializerSettings();
-                settings.Formatting = Formatting.Indented;
-                settings.ContractResolver = new CamelCasePropertyNamesContractResolver();
-
-                settings.DateTimeZoneHandling = DateTimeZoneHandling.Utc;
-                settings.DateFormatHandling = DateFormatHandling.IsoDateFormat;
-
-                return settings;
-            };
-        }
```

Comment:
Would changing the super class of our constructors from `BaseController` to `Controller` mean we might lose these JSON settings? Would we want these settings?

Reply:
As far as I could tell these settings weren't actually implemented.

Initially I tried adding the removal of the null properties here and it had 0 impact on the response.